### PR TITLE
compress world before storing to local storage

### PR DESCRIPTION
**Pull request by chewnoill**

### CODE REVIEW

**Comment by chewnoill** [comment]
blocks | uncompressed | compressed size 
---------|-------|----
 100 | 480K | 80 K 
 500|  2.3M |403 K 
1700 | 7.8 M |  ?
 2.5k | error?  | 2 M

File: app/javascript/components/widgets/numbers/falling-blocks.jsx [hunk 1]
```diff
@@ -337,7 +337,10 @@ class Scene extends React.Component {
 export default compose(
   withApollo,
   graphql(getEventCounts, {
-    options: { variables: { after: getStartOfWeek() } },
+    options: {
+      variables: { after: getStartOfWeek() },
+      fetchPolicy: 'cache-and-network',
+    },
```
Comment (cbarber) [inline app/javascript/components/widgets/numbers/falling-blocks.jsx]: This is handled in the defaultConfig now

File: app/javascript/components/widgets/numbers/panel.jsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ class SubscribedEvents extends React.Component {
     const requestText = pluralize('Request', githubPull);
     const messageText = pluralize('Message', slackMessage);
     return (
-      <div>
+      <div style={{ zIndex: 1, position: 'relative' }}>
```
Comment (cbarber) [inline app/javascript/components/widgets/numbers/panel.jsx]: Prefer styled components

**Comment by cbarber** [comment]
@chewnoill cherry-picked your compress world commit so that is all set.

**Pull request closed without merging**